src/layout/BaseLayout.js

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect, useState } from "react";
+import { Outlet } from "react-router-dom";
+import styles from "../styles/BaseLayout.module.css";
+import { getRequestApi } from "../utils/requestApi";
+import { useRequest } from "../hooks/useRequest";
+
+const Header = () => {
+  const [isSignin, setIsSignin] = useState({});
+
+  const { data, request } = useRequest({
+    url: "api/sample/user",
+    method: "GET",
+  });
+
+  useEffect(() => {
+    if (data) {
+      setIsSignin(data);
+    }
```

Comment:
r: 이곳도 isSignin에 data 객체 대신, data가 있는지 판단하여 true/false값을 넣으면 어떨까요?

```suggestion
    if (data) {
      setIsSignin(true);
      setUser(data);
    }
```